packages/aws-cdk/test/integ/cli/test-cdk-from-outer-directory.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -euo pipefail
+scriptdir=$(cd $(dirname $0) && pwd)
+source ${scriptdir}/common.bash
+# ----------------------------------------------------------
+
+setup
+
+function cleanup() {
+  popd
+  set -e
+}
+
+trap cleanup EXIT INT
+
+pushd ..
```

Comment:
I don't think the cleanup pushd/popd is needed. This script is being executed as a child process.